Cargo.toml

```diff
@@ -36,7 +36,7 @@ workspace = true
 [dependencies]
 vello = { workspace = true }
 thiserror = "1.0.61"
-usvg = "0.42.0"
+usvg = "0.43.0"
 image = { version = "0.25.0", default-features = false, features = [
```

Comment:
I think you need to add `webp` to the features for the `image` crate as well.

Reply:
good catch